wgengine/monitor/monitor_linux.go

```diff
@@ -0,0 +1,50 @@
+// Copyright (c) 2020 Tailscale Inc & AUTHORS All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package monitor
+
+import (
+	"fmt"
+
+	"github.com/mdlayher/netlink"
+	"golang.org/x/sys/unix"
+)
+
+type nlConn struct {
+	conn *netlink.Conn
+}
+
+func NewConn() (*nlConn, error) {
+	conn, err := netlink.Dial(unix.NETLINK_ROUTE, &netlink.Config{
+		// IPv4 address and route changes. Routes get us most of the
+		// events of interest, but we need address as well to cover
+		// things like DHCP deciding to give us a new address upon
+		// renewal - routing wouldn't change, but all reachability
+		// would.
+		//
+		// Why magic numbers? These aren't exposed in x/sys/unix
+		// yet. The values come from rtnetlink.h, RTMGRP_IPV4_IFADDR
+		// and RTMGRP_IPV4_ROUTE.
+		Groups: 0x10 | 0x40,
```

Comment:
add consts for these then, even if they're only func/package-local and unexported. Then put comments there.

Reply:
This was my bad, but thanks for also fixing it wardn :)